smart-contracts/contracts/Unlock.sol

```diff
@@ -183,4 +188,24 @@ contract Unlock is
   {
     return 2;
   }
+
+  // function to read the globalTokenURI field.
+  function getGlobalBaseTokenURI()
+    public
+    view
+    returns (string memory)
+  {
+    return globalBaseTokenURI;
+  }
+
+
+  // function to set the globalTokenURI field.
+  function setGlobalBaseTokenURI(
+    string calldata _URI
+  )
+    external
+    onlyOwner
+  {
+    globalBaseTokenURI = _URI;
```

Comment:
Should we emit an event here?  I'm actually not sure if it's worth it... but I believe we do emit events for all other state changes.

Reply:
Yeah, I think it's good practice in general. I'll add & emit an event.